app/assets/stylesheets/widgets.scss

```diff
@@ -348,8 +353,8 @@
 
 .crayons-layout__comments-display-ad {
   .crayons-sponsorship {
-    .crayons-sponsorship__title {
-      padding-top: var(--su-5);
+    .c-indicator.crayons-sponsorship__indicator {
+      line-height: 150%;
```

Comment:
I needed to add some specificity here so that the line-height specified in the Figma would pull through.